src/Symfony/Component/Process/Process.php

```diff
@@ -1307,7 +1307,7 @@ protected function updateStatus(bool $blocking)
 
         $this->readPipes($running && $blocking, '\\' !== \DIRECTORY_SEPARATOR || !$running);
 
-        if ($this->fallbackStatus && $this->isSigchildEnabled()) {
+        if ($this->fallbackStatus) {
```

Comment:
This is very likely not the right fix. We need to find out why all of a sudden `proc_get_status()` above does not return the expected data on the PHP 8.2 jobs anymore.